src/Tasks/CurlLinkChecker.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace SilverStripe\ExternalLinks\Tasks;
+
+use SS_Cache;
```

Comment:
This is goneskies - see [here](https://docs.silverstripe.org/en/4/changelogs/4.0.0/#cache) for details